frontend/taipy-gui/base/src/exports.ts

```diff
@@ -1,7 +1,7 @@
+import { TaipyApp, createApp, OnChangeHandler, OnInitHandler } from "./app";
 import { WsAdapter } from "./wsAdapter";
-// import { TaipyApp } from "./app";
+import { ModuleData } from "./dataManager";
 
-export {
-    WsAdapter,
-    // TaipyApp,
-};
+export default TaipyApp;
+export { TaipyApp, createApp, WsAdapter };
```

Comment:
why export TaipyApp twice ?

Reply:
I want to have a default export in case they want to use it :D